script.js

```diff
@@ -132,6 +133,12 @@ randomizeBtn.addEventListener('click', randomizeAnswers);
 showListBtn.addEventListener('click', showQuestionsList);
 searchBtn.addEventListener('click', searchQuestions);
 
+searchInput.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    searchQuestions();
+  }
+});
```

Comment:
Nice little user experience detail!

If you're up for the challenge and because I know you seem to enjoy the interaction design then:
Challenge 1: Automatically search on 'keyup'
Challenge 2: Wait 1 second before searching then search because in the real world querying the database all the time can be "expensive". To do this right it would require some digging into the concept of `debounce`.
This is a [good article on someone implementing the search functionality which could be relevant](https://www.geeksforgeeks.org/implement-search-box-with-debounce-in-javascript/).

It's not a part of the homework but if you want an extra challenge!

Reply:
It took some time to figure out how it works and I'm still not sure that I grasp this idea. But I've implemented it. 
Thanks for these challenges!